spring-cloud-aws-messaging/src/test/java/org/springframework/cloud/aws/messaging/config/annotation/SqsConfigurationTest.java

```diff
@@ -133,6 +138,42 @@ void messageHandler_withFactoryConfiguration_shouldUseCustomValues()
 						ConfigurationWithCustomizedMessageHandler.CUSTOM_RESOURCE_ID_RESOLVER);
 	}
 
+
+	@Test
+	void messageHandler_withFactoryConfiguration_shouldUseGlobalDeletionPolicy()
+		throws Exception {
+		// Arrange & Act
+		AnnotationConfigApplicationContext applicationContext = new AnnotationConfigApplicationContext(
+			ConfigurationWithCustomizedMessageHandlerGlobalDeletionPolicy.class);
+		QueueMessageHandler messageHandler = applicationContext
+			.getBean(QueueMessageHandler.class);
+
+		// Assert
+		assertThat(messageHandler.getCustomArgumentResolvers().size()).isEqualTo(1);
+		assertThat(messageHandler.getCustomArgumentResolvers().get(0)).isEqualTo(
+			ConfigurationWithCustomizedMessageHandlerGlobalDeletionPolicy.CUSTOM_ARGUMENT_RESOLVER);
+
+		assertThat(messageHandler.getCustomReturnValueHandlers().size()).isEqualTo(2);
+		assertThat(messageHandler.getCustomReturnValueHandlers().get(0)).isEqualTo(
+			ConfigurationWithCustomizedMessageHandlerGlobalDeletionPolicy.CUSTOM_RETURN_VALUE_HANDLER);
+
+
+		Object sqsMessageDeletionPolicy = ReflectionTestUtils.getField(messageHandler,"defaultSqsMessageDeletionPolicy");
```

Comment:
there should be a space before "defaultSqsMessageDeletionPolicy"

Reply:
Done